Entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
That's the main file and the only one visible in the root folder - rename it to keyclock-user-cli or similar